test_scripts/API/SetAppIcon/010_Mobile_App_received_response_SetAppIcon_REJECTED.lua

```diff
@@ -0,0 +1,68 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0041-appicon-resumption.md
+-- User story:TBD
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+-- Description:
+-- In case:
+-- 1) SDL, HMI are started.
+-- 2) Mobile app is registered. Sends  PutFile and valid SetAppIcon requests.
+-- 3) HMI responds with REJECTED resultCode to SetAppIcon request. Mobile App received response SetAppIcon(REJECTED).
+-- 4) App is re-registered.
+-- SDL does:
+-- 1) Registers an app successfully, responds to RAI with result code "SUCCESS", "iconResumed" = false.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/SetAppIcon/comSetApp')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local requestParams = {
+  syncFileName = "icon.png"
+}
+local requestUiParams = {
+  syncFileName = {
+    imageType = "DYNAMIC",
+    value = common.getPathToFileInStorage(requestParams.syncFileName
+  }
+}
+local allParams = {
+  requestParams = requestParams,
+  requestUiParams = requestUiParams
+}
+
+--[[ Local Functions ]]
+local function setAppIcon_resultCode_REJECTED(params, pAppId)
+  if not pAppId then pAppId = 1 end
+  local mobSession = common.getMobileSession(pAppId)
+  local cid = mobSession:SendRPC("SetAppIcon", params.requestParams)
+  params.requestUiParams.appID = common.getHMIAppId()
+  EXPECT_HMICALL("UI.SetAppIcon", params.requestUiParams)
```

Comment:
Here you expect tat message UI.SetAppIcon will be not received and create Do step for receiving. So ,what step is redundant?